docs/topics/wasm/wasm-js-interop.md

```diff
@@ -1,164 +1,227 @@
 [//]: # (title: Interoperability with JavaScript)
 
-Kotlin/Wasm allows you to both use JavaScript code from Kotlin and Kotlin code from JavaScript.
+Kotlin/Wasm allows you to use both JavaScript code in Kotlin and Kotlin code in JavaScript.
 
-The [Kotlin/JS compiler](js-overview.md) already provides the ability to transpile your Kotlin code to JavaScript. The
-Kotlin/Wasm interoperability with JavaScript is designed in a similar way, taking into account that JavaScript is a
-dynamically typed language compared to Kotlin. Follow our guide to configure interoperability in your projects.
+As with [Kotlin/JS](js-overview.md), the Kotlin/Wasm compiler also has interoperability with JavaScript. If you are 
+familiar with Kotlin/JS interoperability, you will find that Kotlin/Wasm interoperability is similar. However,
```

Comment:
Let's rewrite it to not use the future tense.

Reply:
Changed it for "you can notice".